src/services/fileStorage/SecurityCheckService.js

```diff
@@ -0,0 +1,42 @@
+const { FileModel, SecurityCheckStatusTypes } = require('./model');
+const hooks = require('./hooks/SecurityCheckService.hooks.js');
+const logger = require('../../logger');
+
+const SERVICE_PATH = '/fileStorage/securityCheck/';
```

Comment:
move to ~~configuration file~~ module exports

Reply:
It's used in the module itself, so it can't be moved